arcgis-ios-sdk-samples/Scenes/Surface placements/SurfacePlacementsViewController.swift

```diff
@@ -20,62 +20,116 @@ import ArcGIS
+    }
+    
+    @IBAction func sliderValueChanged(_ sender: UISlider) {
+        let zValue = Double(sender.value)
+        zValueLabel.text = zValueFormatter.string(from: Measurement<UnitLength>(value: zValue, unit: UnitLength.meters))
+        // Set the z-value of each geometry of surface placement graphics.
+        overlaysBySurfacePlacement.values.forEach { graphicOverlay in
+            graphicOverlay.graphics.forEach { graphic in
+                let g = graphic as! AGSGraphic
+                g.geometry = AGSGeometryEngine.geometry(bySettingZ: zValue, in: g.geometry!)
+            }
+        }
+    }
+    
+    // MARK: Initialize scene and make graphics overlays
+    
+    /// Create a scene.
+    ///
+    /// - Returns: A new `AGSScene` object.
+    func makeScene() -> AGSScene {
+        let scene = AGSScene(basemap: .imagery())
+        // Add a base surface for elevation data.
         let surface = AGSSurface()
-        /// The url of the Terrain 3D ArcGIS REST Service.
+        // Create elevation source from the Terrain 3D ArcGIS REST Service.
         let worldElevationServiceURL = URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!
         let elevationSource = AGSArcGISTiledElevationSource(url: worldElevationServiceURL)
         surface.elevationSources.append(elevationSource)
+        // Create scene layer from the Brest, France scene server.
+        let sceneServiceURL = URL(string: "https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer")!
+        let sceneLayer = AGSArcGISSceneLayer(url: sceneServiceURL)
         scene.baseSurface = surface
-        
-        super.init(coder: coder)
+        scene.operationalLayers.add(sceneLayer)
+        return scene
     }
     
-    /// The scene view managed by the view controller.
-    @IBOutlet var sceneView: AGSSceneView!
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        self.sceneView.scene = scene
-        
-        let camera = AGSCamera(latitude: 53.05, longitude: -4.01, altitude: 1_115, heading: 299, pitch: 88, roll: 0)
-        self.sceneView.setViewpointCamera(camera)
-        
-        let graphicsOverlays = [
-            makeGraphicsOverlay(surfacePlacement: .drapedBillboarded),
-            makeGraphicsOverlay(surfacePlacement: .relative),
-            makeGraphicsOverlay(surfacePlacement: .absolute)
-        ]
-        sceneView.graphicsOverlays.addObjects(from: graphicsOverlays)
-        
-        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["SurfacePlacementsViewController"]
-    }
-    
-    /// Creates a graphics overlay for the given surface placement.
+    /// Create a graphics overlay for the given surface placement.
     ///
-    /// - Parameter surfacePlacement: The surface placement for which to create
-    /// a graphics overlay.
+    /// - Parameters:
```

Comment:
I recommend using the format generated by Xcode, which is what was already there:

![image](https://user-images.githubusercontent.com/2257493/87206657-1845ef80-c2bf-11ea-9f20-014c6c218ba8.png)

The rule seems to be `- Parameter` for a single parameter and `- Parameters:` for multiple parameters.

Reply:
Ha I see... The comment formats for single and multiple params are different! Will fix.